shared/src/main/scala/drt/shared/Api.scala

```diff
@@ -299,71 +305,46 @@ case object VoyageNumber {
   }
 }
 
-case class Arrival(
-                    Operator: Option[String],
-                    Status: String,
-                    Estimated: Option[MillisSinceEpoch],
-                    Actual: Option[MillisSinceEpoch],
-                    EstimatedChox: Option[MillisSinceEpoch],
-                    ActualChox: Option[MillisSinceEpoch],
-                    Gate: Option[String],
-                    Stand: Option[String],
-                    MaxPax: Option[Int],
-                    ActPax: Option[Int],
-                    TranPax: Option[Int],
-                    RunwayID: Option[String],
-                    BaggageReclaimId: Option[String],
-                    AirportID: PortCode,
-                    Terminal: Terminal,
-                    rawICAO: String,
-                    rawIATA: String,
-                    Origin: PortCode,
-                    Scheduled: MillisSinceEpoch,
-                    PcpTime: Option[MillisSinceEpoch],
-                    FeedSources: Set[FeedSource],
-                    CarrierScheduled: Option[MillisSinceEpoch] = None,
-                    ApiPax: Option[Int] = None
+case class Operator(code: String)
+
+case class ArrivalStatus(description: String) {
+  override def toString: String = description
+}
+
+case class Arrival(Operator: Option[Operator],
+                   CarrierCode: CarrierCode,
+                   VoyageNumber: VoyageNumber,
+                   Status: ArrivalStatus,
+                   Estimated: Option[MillisSinceEpoch],
+                   Actual: Option[MillisSinceEpoch],
+                   EstimatedChox: Option[MillisSinceEpoch],
+                   ActualChox: Option[MillisSinceEpoch],
+                   Gate: Option[String],
+                   Stand: Option[String],
+                   MaxPax: Option[Int],
+                   ActPax: Option[Int],
+                   TranPax: Option[Int],
+                   RunwayID: Option[String],
+                   BaggageReclaimId: Option[String],
+                   AirportID: PortCode,
+                   Terminal: Terminal,
+                   Origin: PortCode,
+                   Scheduled: MillisSinceEpoch,
+                   PcpTime: Option[MillisSinceEpoch],
+                   FeedSources: Set[FeedSource],
+                   CarrierScheduled: Option[MillisSinceEpoch],
+                   ApiPax: Option[Int]
```

Comment:
Are you missing an update to protobuf in here?

Reply:
I've left the protobuf message as is. It just stores the same flight code for both the iata & icao. I guess technically I should update it :)